requirements.txt

```diff
@@ -1,6 +1,6 @@
 requests>=2.10.0
 uritemplate>=3.0.0
 promise>=1.0.1
-marshmallow>=3.19.0
+marshmallow>=3.19.0,<3.24.0
```

Comment:
There is a breaking change in [Marshmallow 3.24](https://marshmallow.readthedocs.io/en/latest/changelog.html#id8). We need to stop using fields.Number according to that change. I am not sure at the moment how that might affect us, so we'll look at that later.

Reply:
It looks like it was just deprecated in 3.24 and not breaking until 4.0 where it's actually removed.  We've been using 3.26.1 for the past 2 months and haven't noticed any issues.  Nice catch and callout, though!

Reply:
ah, ok.  I thought maybe it was just based on the changelog so wanted to point that out.  It's weird that it broke the tests when it definitely shouldn't have (ie. it should have just been a deprecation and not a breaking change on Marshmallow's end).

When can we expect a release?

Reply:
Released in 4.6.0